plugins/BEdita/Core/src/Command/FixHistoryCommand.php

```diff
@@ -227,7 +227,7 @@ protected function fixHistoryUpdate(ObjectEntity $object): void
      */
     protected function historyEntity(ObjectEntity $object): History
     {
-        $history = $this->History->newEntity();
+        $history = $this->History->newEntity([]);
```

Comment:
To make happy scrutinizer:

```suggestion
       /** @var \BEdita\Core\Model\Entity\History $history */
        $history = $this->History->newEntity([]);
```

Reply:
done